centos_ci.sh

```diff
@@ -220,7 +219,7 @@ else
   source ~/jenkins-env # Source environment variables for minishift_ci user
   PASS=$(echo $CICO_API_KEY | cut -d'-' -f1-2)
 
-  if [[ "$JOB_NAME" = "minishift-docs" ]]; then
+  if [[ "$JOB_NAME" != "minishift-docs" ]]; then
```

Comment:
Intended as I need to check that path, will remove it once I am convinced it is fine.